internal/utils/utils.go

```diff
@@ -7,16 +7,23 @@ import (
 	"github.com/Issif/falco-talon/internal/event"
 )
 
+const (
+	errorType    = "error"
```

Comment:
if you use `terminateStr` it seems more relevant to have `errorStr`, wdyt?

Reply:
done